README.md

```diff
@@ -0,0 +1,3 @@
+# cluster-baremetal-operator
+This repository is for a new second level operator for baremetal installs. 
+See https://github.com/openshift/enhancements/pull/212 for details about this operator.
```

Comment:
We should link to the published doc, since future PRs may update it.

```suggestion
See https://github.com/openshift/enhancements/blob/master/enhancements/baremetal/an-slo-for-baremetal.md for details about this operator.
```